src/proposals/Proposals.jl

```diff
@@ -371,7 +371,7 @@ function (prop::Slice)(rng::AbstractRNG,
     local idxs, r, u, u_prop, v_prop, logl_prop, logl_l, logl_r
     
     # modifying axes and computing lengths
-    axes, axlens = Bounds.decompose(bounds)
+    axes = Bounds.decompose(bounds)
```

Comment:
that's not how that works- the `decompose` function returns both, so now `axes` is a Tuple instead of the matrix we want. You'll need to change to use `Bounds.axes(bounds)`